spec/helpers/visit_helper_spec.rb

```diff
@@ -60,6 +60,17 @@ def visit
       expect(helper.prison_address).to start_with "1 Fort Road"
     end
 
+    context "with illegal html in the address" do
+      before :each do
+        expect(Rails.configuration.prison_data['Rochester']).to receive(:[]).
```

Comment:
This should probably be an `allow` as it's not the thing we're testing but is just configuration of the test environment. Personally, I'd say don't put expectations in `before` or `after` blocks and aim to have only one per spec (feature specs excepted).